templates/navigation.hbs

```diff
@@ -1,6 +1,9 @@
 {{! Maintainers - Daryl Hedley}}
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link a11y-ignore a11y-ignore-focus" data-event="skipNavigation" tabindex="0" aria-label="{{_globals._accessibility._ariaLabels.skipNavigation}}">{{{_globals._accessibility.skipNavigationText}}}</button>
```

Comment:
Please remove `tabindex="0"` that seems to have crept in and we should be good to go.